qemu/tests/cfg/uefi_check_boot_option.cfg

```diff
@@ -0,0 +1,36 @@
+- uefi_check_boot_option:
+    only q35
+    only ovmf
+    type = uefi_check_boot_option
+    start_vm = no
+    boot_menu = on
+    del boot_once
+    del boot_order
+    del boot_strict
+    variants:
+        - splash_time_10:
+            boot_splash_time = 10000
+            splash_time_pattern = "SetVariable\(Timeout,\s%d\)"
+        - splash_time_12:
+            boot_splash_time = 12000
+            splash_time_pattern = "SetVariable\(Timeout,\s%d\)"
```

Comment:
You don't need to define it twice. Move it to Line 9.

Reply:
Yes, it has been modified, thank you for your patience.